web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/RevurderingJson.kt

```diff
@@ -176,6 +180,7 @@ internal data class UnderkjentRevurderingJson(
     val fritekstTilBrev: String,
     val skalFøreTilBrevutsending: Boolean,
     val informasjonSomRevurderes: Map<Revurderingsteg, Vurderingstatus>,
+    val simuleringForAvkortingsvarsel: SimuleringJson?,
```

Comment:
@aryani1 @RamziAbuQassim Gir det mer mening om den vanlige simuleringen og den for avkortinga ligger inne i samme object?